### ISSUE TITLE
The displaced fluid scale should use the Scale Readout Node

### ISSUE THREAD

**Issue by AgustinVallejo** [phetsims/buoyancy#169]
The readout on the scale under the fluid displaced beaker should use ScaleReadoutNode. Was going to do it but I'm not very familiar with that implementation. Should be very straightforward by pairing.

https://github.com/phetsims/density-buoyancy-common/blob/bc99fa90b200870b2473a606fbb0c311b8dfb599/js/buoyancy/view/FluidDisplacedAccordionBox.ts#L148-L154

**Comment by samreid** [comment]
Yes, @zepumph recommends using the same ScaleReadoutNode.

**Comment by zepumph** [comment]
I think I prefer the commit I just made, but here are the tradeoffs:

* It seems the main benefit of the commit is adding support for 1/2 decimal places, and the duplication of the pattern string Property.
* It wasn't a good fit to have the blendedNumberProperty running this scale, so I had to create a subtype that allowed for this usage to not need a `Scale` instance to use. 

Let me know what you think!
